tests/unit_tests/test_dqc_us_0005.py

```diff
@@ -128,6 +128,68 @@ def test_axis_exists_multi_axis(self):
             facts.axis_exists(mock_val, mock_fact, 'not a used axis')
         )
 
+    def test_run_checks_EntityCommonStockSharesOutstanding(self):
+        msg = messages.get_message('DQC.US.0005', "17")
+        mock_context = Mock(endDatetime=1)
+        fact = Mock(localName='EntityCommonStockSharesOutstanding', context=mock_context)
+        lookup = 'foo'
+        eop_results = {lookup:[1, 1]}
+        mock_error = Mock()
+        mock_modelxbrl = Mock(error=mock_error)
+        mock_val = Mock(modelXbrl=mock_modelxbrl)
+        dqc_us_0005.run_checks(mock_val, fact, eop_results, lookup)
+        self.assertFalse(mock_error.called)
+
+        mock_context=Mock(endDatetime=0)
+        fact = Mock(localName='EntityCommonStockSharesOutstanding', context=mock_context)
+        dqc_us_0005.run_checks(mock_val, fact, eop_results, lookup)
+        mock_error.assert_called_with(
+            'DQC.US.0005.17',
+             msg,
+            modelObject=[fact] + list(eop_results[lookup]),
+            ruleVersion=dqc_us_0005._RULE_VERSION
+        )
+
+    @patch('dqc_us_rules.dqc_us_0005.facts.axis_exists')
+    def test_run_checks_axis_exists(self, axis_exists):
+        axis_exists.return_value = True
```

Comment:
SG: Single line testing docstring missing. ([REF](https://github.com/Workiva/Stylin/blob/master/STYLEGUIDE.rst#docstrings-and-unit-tests))